cmd/bundle/run.go

```diff
@@ -68,6 +70,14 @@ func newRunCommand() *cobra.Command {
 		}
 
 		runOptions.NoWait = noWait
+		if restart {
+			cmdio.LogString(ctx, "Cancelling the run...")
+			err := runner.Cancel(ctx)
+			if err != nil {
+				return err
+			}
+			cmdio.LogString(ctx, "All runs have been cancelled, starting a new run")
```

Comment:
We could turn both of these into a spinner; then it won't show once cancellation has completed.